cmd/application/save.go

```diff
@@ -65,8 +65,12 @@ func saveApplication(cmd *cobra.Command, options SaveOptions) error {
 		return err
 	}
 	initialApp, err := util.ParseJsonFromFileOrStdin(options.applicationFile)
+	if err != nil && strings.HasPrefix(err.Error(), "No json input") {
```

Comment:
I wonder if it would be more robust to pass in a flag to `ParseJsonFromFileOrStdin` controlling whether to error or just return `nil` if there's nothing in the file or stdin.  (Looks like there are a couple of places where we're looking for that particular error string and clearing it out.)

Reply:
Probably makes sense. Maybe even another function.